proto/ecdhes_aead.proto

```diff
@@ -0,0 +1,92 @@
+// Copyright 2017 Google Inc.
```

Comment:
why do we have this directory at the top level? these are protos for tink.

Reply:
I will move it.

Reply:
moved under `protos/tink`